questions.md

```diff
@@ -4,26 +4,54 @@
 
 1. On the UCI cluster, the resource request "-pe openmp 64" refers to the number of processors requested.  Does that
    request mean that your commands will use multiple processors?
+   #### Using the command '-pe openmp 64' just reserves 64 processors for your job, it doesn't mean that your job will actually use that many processors. The number of processors used by your job is determined by the actual script and parameters within it.
 2. In general, how do you know how many processors, how much RAM, how many files would be required/needed/written by the
    jobs you are running on the cluster?
+   #### If you haven't run a job before you can run a test job with sample inputs along with the following line of code before your commands: 
+   <pre><code>/usr/bin/time -v</pre></code>
+   #### This will output CPU time, RAM, and files required/needed/written requirements for your command.
```

Comment:
this is good but it is is also good to add option -o which allows an output file. env time -v -o MYOUTPUT CMD would do the trick and do a dump to a text file. this helps sometimes if you care about what goes in your stdout and stderr files. I like to keep it seperate as I usually do tail -n 1 MYSTDOUTFILE to see if I was able to run to completion and then sort my STDERRFILES using ls -lS jobname.ejobid* by size to see if the stderr produced larger output than normal. this typically means there where errors on the run. This doesn't work though when the programmers are sloppy and lazy and output stdout to stderr. :( You could also look at exit status in your outputfile to see the if it ran successfully or not. This is important if you have several commands running within your qsub script.